apps/ui-builder/src/components/ComponentTree.tsx

```diff
@@ -189,32 +103,23 @@ export const ComponentTree: React.FunctionComponent<ComponentTreeProps> = ({
   );
 
   const selectedComponentId = selectedComponent?.uuid as string;
-  const items: TreeItemProps[] =
-    tree.props?.children?.map(item =>
-      jsonTreeToTreeItems(
-        item,
-        selectedComponentId,
-        handleSelectComponent,
-        handleClone,
-        handleMove,
-        handleDeleteSelected,
-        handleAddComponent,
-        handleDeleteComponent,
-      ),
-    ) ?? [];
-  items.forEach(item => getActiveItemIds(item));
+  const flat = treeToArray(tree, 0);
 
   return (
-    <Tree
```

Comment:
not sure if removing Tree is a good idea. I can see you added arrow navigation, however:
- arrow left should close subtree and/or navigate to parent
- arrow right should open subtree and navigate to first child
- letter keys should be used to jump to items starting with particular letters
- HTML roles are not present so screen reader users will not understand what kind of UI they are interacting with.

It is quite tricky to implement Tree fully - so I would say we should stick to v0 tree and wait for v9 Tree to be built first.

Reply:
Reimplemented to use Tree and it's features